index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```

Comment:
- [ ] Each HTML page must include a reference to the external style sheet file inside the <link> element, inside the head section. Kindly add this line to link the CSS file to the HTML file.

```
<link rel="stylesheet" href="style.css"> 
```